### ISSUE TITLE
united_kingdom_contracts_finder_records: 404 (2024-10)

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/kingfisher-collect#1115]
All the /Record/ URLs seem to 404

e.g. https://www.contractsfinder.service.gov.uk/Published/Notice/records/ocds-b5fd17-373a795e-2bc3-43c3-82a7-e8d7bbb1490c

```json
{
    "code": "ValidationError",
    "errors": 3200
}
```

The registry uses _releases instead.

**Comment by yolile** [comment]
You are right, I've only tried with the link in the issue, restored [now](https://github.com/open-contracting/kingfisher-collect/commit/008c60c336a3bc82e3e2da5df74d7df8ace74a44)
